src/ucp/rndv/rndv.c

```diff
@@ -577,6 +578,10 @@ UCS_PROFILE_FUNC_VOID(ucp_rndv_get_completion, (self), uct_completion_t *self)
         return;
     }
 
+    ucs_assertv(rndv_req->send.state.uct_comp.count == 0,
```

Comment:
maybe add common ucp helper which calls completion if it reached 0, and use it in relevant places (ff and progress)?
otherwise, better check ==0 right in the beginning of the function

Reply:
done

Reply:
I think the assertion comp count==0 can be removed now?

Reply:
do we still need to add this assert?

Reply:
> do we still need to add this assert?

removed